gptscript.go

```diff
@@ -307,6 +307,60 @@ func (g *GPTScript) PromptResponse(ctx context.Context, resp PromptResponse) err
 	return err
 }
 
+func (g *GPTScript) ListCredentials(ctx context.Context, credCtxs []string, allContexts bool) ([]Credential, error) {
```

Comment:
Can credCtxs and allContext be added to an Options struct.  If no options are passed is should do the same thing as `gptscript creds` which I really don't know if that is "all contexts" or what.

Reply:
Will do

Reply:
Done